src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -87,8 +87,17 @@
                                 <Rectangle Style="{StaticResource ExpanderSeparatorStyle}" />
                                 <CheckBox x:Uid="FancyZones_ShowZonesOnAllMonitorsCheckBoxControl" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.ShowOnAllMonitors}" Margin="{StaticResource ExpanderSettingMargin}"  />
                                 <Rectangle Style="{StaticResource ExpanderSeparatorStyle}" />
-                                <CheckBox x:Uid="FancyZones_SpanZonesAcrossMonitorsCheckBoxControl" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.SpanZonesAcrossMonitors}" Margin="{StaticResource ExpanderSettingMargin}"  />
-                               
+                                <CheckBox IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.SpanZonesAcrossMonitors}" Margin="{StaticResource ExpanderSettingMargin}" >
```

Comment:
Is the custom control needed for setting the IsEnabled state (+brushes) or?

Reply:
I was thinking between a custom control and a simple `TextBlock` with converter `IsEnabled` -> `Foreground`. Is the second approach better?

Reply:
No, this is fine - just wanted to make sure :). It's actually useful to have for other labels in the future as well!